lib/noun-project-api/icon.rb

```diff
@@ -1,35 +1,36 @@
 module NounProjectApi
+  # A single Icon as an abstracted ruby object.
   class Icon
     PREVIEW_SIZE_200 = 200
     PREVIEW_SIZE_42 = 42
     PREVIEW_SIZE_84 = 84
 
-    PUBLIC_DOMAIN_LICENSE = "public-domain"
+    PUBLIC_DOMAIN_LICENSE = 'public-domain'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.